src/model/hint.js

```diff
@@ -0,0 +1,66 @@
+import _ from 'lodash';
+
+//import { ENTITY_TYPE } from './entity-type.js';
+
+const HINT_TYPE = Object.freeze({
+    ORGANISM: 'organism'
+});
+const HINT_TYPES = _.flatMap(HINT_TYPE);
+
+
+const HINT_PASSAGE = Object.freeze({
+    TITLE: 'title',
+    ABSTRACT: 'abstract'
+});
+const HINT_PASSAGES = _.flatMap(HINT_PASSAGE);
+
+class Hint{
+
+    get text(){
```

Comment:
What is this field: (a) the entire text (b) the mention (c) other?

Reply:
(b) the mention + added comments to clarify.